### PR TITLE
Add latest application and normal conf. Vespa, and blockers, to dep.g…

### PR DESCRIPTION
**Pull request by jonmv**
…raph.view API

@smorgrav please review and merge.

### CODE REVIEW

File: controller-server/src/test/java/com/yahoo/vespa/hosted/controller/restapi/application/responses/deployment-overview.json [hunk 1]
```diff
@@ -6,19 +6,47 @@
...
+          "blockers": []
+        },
+        "application": {
+          "application": {
+            "build": 4,
+            "compileVersion": "6.1.0",
+            "sourceUrl": "repository1/tree/commit1",
+            "commit": "commit1"
+          },
+          "at": 1000,
```
Comment (smorgrav) [inline controller-server/src/test/java/com/yahoo/vespa/hosted/controller/restapi/application/responses/deployment-overview.json]: What is this?

File: controller-server/src/test/java/com/yahoo/vespa/hosted/controller/restapi/application/responses/deployment-overview.json [hunk 2]
```diff
@@ -6,19 +6,47 @@
...
+          "upgrade": false,
+          "blockers": []
```
Comment (smorgrav) [inline controller-server/src/test/java/com/yahoo/vespa/hosted/controller/restapi/application/responses/deployment-overview.json]: Do you have a test where there are blockers?

File: controller-server/src/test/java/com/yahoo/vespa/hosted/controller/restapi/application/responses/deployment-overview.json [hunk 3]
```diff
@@ -6,19 +6,47 @@
...
+        },
+        "application": {
+          "application": {
+            "build": 4,
+            "compileVersion": "6.1.0",
+            "sourceUrl": "repository1/tree/commit1",
+            "commit": "commit1"
+          },
+          "at": 1000,
+          "upgrade": false,
```
Comment (smorgrav) [inline controller-server/src/test/java/com/yahoo/vespa/hosted/controller/restapi/application/responses/deployment-overview.json]: And this upgrade field - what is that?

**Review by smorgrav — COMMENTED** [review]
So just some clarifications for a tired frontend developer and this looks good to me. Maybe except an example that contains serialization of blokkers

File: controller-server/src/test/java/com/yahoo/vespa/hosted/controller/restapi/application/responses/deployment-overview.json (hunk 1 above)
Comment (jonmv) [reply controller-server/src/test/java/com/yahoo/vespa/hosted/controller/restapi/application/responses/deployment-overview.json]: millis at which the version was built (application) or of last commit (Vespa).

File: controller-server/src/test/java/com/yahoo/vespa/hosted/controller/restapi/application/responses/deployment-overview.json (hunk 2 above)
Comment (jonmv) [reply controller-server/src/test/java/com/yahoo/vespa/hosted/controller/restapi/application/responses/deployment-overview.json]: I’ll add some.

File: controller-server/src/test/java/com/yahoo/vespa/hosted/controller/restapi/application/responses/deployment-overview.json (hunk 3 above)
Comment (jonmv) [reply controller-server/src/test/java/com/yahoo/vespa/hosted/controller/restapi/application/responses/deployment-overview.json]: Whether to show the “upgrade now” button - hide it when no prod deployment in the instance are on a lower version than this version.

**Comment by jonmv** [comment]
There we go.

**Pull request merged**